src/components/pages/Friends/Friends.js

```diff
@@ -1,11 +1,72 @@
 import React from 'react';
+import smashRequests from '../../../helpers/data/smashRequests';
+import authRequests from '../../../helpers/data/authRequests';
+import FriendItem from '../../FriendItem/FriendItem';
 import './Friends.scss';
 
 class Friends extends React.Component {
+  state = {
+    users: [],
+    potentials: [],
+    confirmed: [],
+    pending: [],
+  }
+
+  componentDidMount() {
+    this.getAndSortUsers();
+  }
+
+  getAndSortUsers = () => {
+    const uid = authRequests.getCurrentUid();
+    smashRequests
+      .usersAndFriends(uid)
+      .then((results) => {
+        const users = results;
+        const potentials = users.filter(user => !user.isAccepted && !user.isPending);
+        const pending = users.filter(user => !user.isAccepted && user.isPending);
+        const confirmed = users.filter(user => user.isAccepted);
+        this.setState({
+          users,
+          potentials,
+          pending,
+          confirmed,
+        });
+      })
+      .catch(err => console.error('error in SMASH', err));
+  }
+
   render() {
+    const {
+      potentials,
+      pending,
+      confirmed,
+    } = this.state;
+
+    const friendItemComponents = friendArray => (
+      friendArray.map(friend => (
+        <FriendItem key={friend.id} friend={friend} />
+      ))
+    );
+
     return (
-      <div className='Friends'>
-        <h2>Friends Component</h2>
+      <div className='Friends container'>
+        <h2>Friends</h2>
+        <div className="container">
+          <div className="row">
+            <div className="col-sm">
+              <h3>Potential Friends</h3>
+              <ul>{friendItemComponents(potentials)}</ul>
```

Comment:
friendItemComponents will eventually need more information - ie what kind of friend is it - so you know what buttons to display.  i am assuming you are going to do this in another PR when you make those buttons but just reminding you